TOOLING.md

```diff
@@ -0,0 +1,15 @@
+# Tooling
+
+This documents describes at a high level some the key tools used across `near-js`.
+
+## Package Manager - [PNPM](https://pnpm.io/)
+PNPM is a package manager focussed on fast installations while still be disk efficient. It manages dependency installations across the monorepo as well as linking interdependent packages. Unlike other package managers, PNPM has a non-flat `node_modules` structure which ensures packages only have access to dependencies defined in their `package.json`.
+
+## Build System - [Turborepo](https://turborepo.org/)
+Turborepo provides tooling for orchestrating commands across the monorepo. Similar to Yarn Workspaces, Turbo can run tasks within each project via a single command, but rather than executing serially, Turbo has various performance enhancing features such as task parallelization and local/remote caching.
```

Comment:
Are `Turborepo` and `Turbo` interchangeable or should it be referred to as `Turborepo` throughout?

Reply:
They are (the CLI is `turbo`) but I'll use `Turborepo` exclusively here to be consistent.